contexts/did-v1.jsonld

```diff
@@ -11,6 +11,8 @@
     "EcdsaSecp256k1VerificationKey2019": "sec:EcdsaSecp256k1VerificationKey2019",
     "Ed25519Signature2018": "sec:Ed25519Signature2018",
     "Ed25519VerificationKey2018": "sec:Ed25519VerificationKey2018",
+    "JsonWebKey2020": "sec:JsonWebKey2020",
```

Comment:
Is the purpose of JsonWebKey2020 to express private key material? Why does it deviate from other LD Security key formats by removing the "Verification" nomenclature? I'm asserting that doing so is dangerous. Can we change the name to this:

* JwkVerificationKey2020

If not, are there other key names that would be acceptable to the JWK community?

Reply:
I'd be open to renaming it to be JwkPublicKey2020, but last I heard your experience has taught you developers don't know what a public key is and verification key was more helpful. However, I'm opposed to defining the key usage within the key suite because it ends up coupling key types and LD cryptographic suites together in a way that makes in cumbersome to use, support, and define new suites.

The purpose in defining them this way is to allow this key suite which defines only a representation to be referenced by any cryptographic suite.

Additionally, if the intent behind verification key is to express key usage, why don't we add a key usage property the same way it's been done in web crypto and JOSE? 

I'm not understanding how this coupling gets us anything. This is why I've advocated that we separate the keys from the cryptographic suites and explicitly document the dangers within the key suites and cryptographic suites around non best practice usage (auto conversion from Ed25519 to X25519) which has already been seen when using ld proof suites. 

Furthermore, this coupling is actually more of a nice to have naming scheme rather than preventing the actual danger of key reuse.

As an example, nothing would stop me from taking a public key used in something like RSAVerificationKey2019 and reusing it in RSAEncryptionKey2020 which if the wrong algorithm choices are made then signatures can be forged via a Bleichenbacher’s attack. 

This is all to say, I really don't think we're buying ourselves safe patterns here by combining the signature and the key usage into a single suite and I know from experience that implementing this is far more cumbersome because of it which is why I'd advocate for this to not include the key usage in the name of the cryptographic suite.